### DISCUSSION TITLE
Bias-T

### DISCUSSION THREAD

**Discussion by damiconi** [jekhokie/raspberry-noaa-v2#562] [Q&A]
Hi
 just a question. I've a RTL_SDR V3 dongle with a LNA. If in line  "noaa_15_enable_bias_tee: false" of setting.yml  I change false in true the bias-t of my dongle is on?
Thanks so much

Nico

**Comment by L-A-H** [comment]
I'm not a programmer per se but since this is more of a hardware related subject I will give it a shot.

My guess would be yes, since the SW does not know if you have an LNA on the signal chain it would do what you tell it to do ie enable the bias-t. 

However if you are simultaneously powering the lna externally together with the bias-t, this could have some undesirable results. In my understanding this would be similar to connecting two power sources in parallel and would have different consequences depending on the nature of the connected sources. I would not advise connecting 2 switching power supplies in parallel as could be fatal for both devices. A non rechargeable battery with a switching supply could end up catastrophic for the battery.

Thinking about bias-t, I wonder if the bias voltage is activated only during a pass or continuously.
